File: Pracka.Cup.API/Models/CreateHistoryDto.cs
namespace Pracka.Cup.API.Models
{
    using Pracka.Cup.Database.Enums;
    using System;

    public class CreateHistoryDto
    {
        public int HomeTeamId { get; set; }
        public int PlayerHomeTeamId { get; set; }
        public int GoalsHomeTeam { get; set; }

        public int AwayTeamId { get; set; }
        public int PlayerAwayTeamId { get; set; }
        public int GoalsAwayTeam { get; set; }

        public GameTypeEnum GameType { get; set; }
        public DateTime? GameDateUTC { get; set; }
    }
}

File: Pracka.Cup.Web/src/api/models/create-history-dto.ts
import { GameType } from "./game-type-enum";

export type CreateHistoryDto = {
    homeTeamId: number
    playerHomeTeamId: number
    goalsHomeTeam: number

    awayTeamId: number
    playerAwayTeamId: number
    goalsAwayTeam: number

    gameDateUTC: Date
    gameType: GameType
}